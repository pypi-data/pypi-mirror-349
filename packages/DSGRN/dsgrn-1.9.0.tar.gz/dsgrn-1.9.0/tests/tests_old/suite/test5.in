#!/bin/bash
$1 network networks/network2.txt wallgraph json 43

