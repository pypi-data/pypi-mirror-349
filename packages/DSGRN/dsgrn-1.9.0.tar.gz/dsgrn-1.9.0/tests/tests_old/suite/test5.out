[[2],[0,4],[3],[7],[7],[7],[5],[7]]
